data/json/npcs/talk_tags.json

```diff
@@ -748,7 +958,11 @@
       "This sucks.  I'm too big.",
       "I don't like being a mutant crammed into your tiny little vehicle.",
       "Getting awfully cramped in here.",
-      "I don't think this vehicle was designed for someone like me."
+      "I don't think this vehicle was designed for someone like me.",
+      "Was this thing designed for ants or something?",
+      "I feel like this vehicle wasn't build for overgrown freaks in mind.",
```

Comment:
`"I feel like this vehicle wasn't build with overgrown freaks in mind.",`

Reply:
"I feel like this vehicle wasn't _built_ with overgrown freaks in mind."

I'll change this one 👍